### ISSUE TITLE
Unable to unverify (and update) collection of NFT

### ISSUE THREAD

**Issue by serejke** [metaplex-foundation/js#431]
**Use case**: I want to unverify and update the collection of an NFT. In my case the old collection is not sized.
I call the following builder and send transaction

```
const transactionBuilder = this.metaplex.nfts().builders().update(
      {
        nftOrSft: nft,
        updateAuthority,
        collectionAuthority,
        oldCollectionIsSized: false,
        collection: null
      }
```

**Problem:**
The program throws an error about invalid collection update authority
 
```
> Program logged: "Instruction: Unverify Collection"
> Program logged: "Collection Update Authority is invalid"
> Program consumed: 10015 of 600000 compute units
> Program returned error: "custom program error: 0x51"
```

This is because of this line:
https://github.com/metaplex-foundation/js/blob/c171e1e31d9fe12852afb39e449123339848180e/packages/js/src/plugins/nftModule/operations/updateNft.ts#L316

Metaplex-JS sets the collection authority to be `updateAuthority` of the NFT itself, which has nothing to do with the collection's authority. So this should be changed to `params.collectionAuthority` instead.

**Comment by lorisleiva** [comment]
Hi there 👋 

Thanks for raising this.

I've added a new optional attribute to this operation so you can provide a custom `oldCollectionAuthority` as a `Signer` instead of the default `updateAuthority`.

I couldn't use the existing `collectionAuthority` attribute because that's referring to the new collection, not the current one.

This has been merged and published in version `0.17.10`.

Let me know if that fixes your issue.

**Comment by serejke** [comment]
@lorisleiva thank you for the-same-day fix! It has solved my issue.
